### ISSUE TITLE
Nearest Neighbor search on Word Embeddings 

### ISSUE THREAD

**Issue by raman-r-4978** [carschno/JFastText#9]
Hi,

Just wanted to know that Is it possible to execute this ```./fasttext nn model.bin``` command, via JAVA call?

[reference](https://fasttext.cc/docs/en/unsupervised-tutorial.html#nearest-neighbor-queries)

**Comment by carschno** [comment]
Yes, `nn` should work, although it is not really tested. Have you tried it?

**Comment by raman-r-4978** [comment]
Thanks for the reply

Yes it is possible, when I execute 
```
jft.runCmd(
	new String[]{
	    "nn",
    	    "model.bin"
    }
);
```

By default `nn` takes input from `stdin`. [Check here](https://fasttext.cc/docs/en/unsupervised-tutorial.html#nearest-neighbor-queries)

Is there any way to bring this functionality into Java. Something like this,
```
jft.loadModel("model.bin");
List<NearestNeighbors> nn = jft.getNearestNeighbors("sampleWord", 10);
```

Hope you get me now.
Thanks

**Comment by carschno** [comment]
I'm afraid this feature is not implemented indeed. You [mentioned](https://github.com/vinhkhuc/JFastText/issues/12#issuecomment-665423819) you had a fork implementing; feel free to create a pull request from it!

**Comment by raman-r-4978** [comment]
I have no experience with bindings and JNI. Could you give some tips on this? That would be helpful for me to develop this feature.

Thanks
